runtime/queries/svelte/highlights.scm

```diff
@@ -0,0 +1,68 @@
+; Special identifiers
```

Comment:
Some of these need to be converted into Helix's scopes: https://docs.helix-editor.com/themes.html

Reply:
@archseer Scopes look fine to me now; do we want to also add the `none` scope?